src/components/Home/Landing/Modal/StarPrizeModal/StarPrizeModal.tsx

```diff
@@ -28,20 +30,24 @@ function StarPrizeModal({ isHabitStart, blueText, comment, yellowText }: StarPri
 		<Modal>
 			<div css={containerStyle} onClick={() => dispatch(closeModal())}>
 				<span css={imageWrapperStyle}>
-					<img src={StarImage} alt="prizeImage" />
+					{isHabitStart ? (
+						<Lottie animationData={StarMedalAnimation} loop={false} />
+					) : (
+						<Lottie animationData={ClapAnimation} loop={false} />
+					)}
```

Comment:
제 생각에는 Lottie 컴포넌트의 animationData만 다르고 나머지는 전부 동일해서 
아래처럼 animationData에 바로 조건에 따라 값을 달리 줘도 될 것 같은데, 혹시 이렇게 작성하신 이유가 있을까요? 
```suggestion
					<Lottie animationData={isHabitStart ? StarMedalAnimation : ClapAnimation} loop={false} />
```